packages/twenty-ui/src/utilities/device/__tests__/getOsShortcutSeparator.test.ts

```diff
@@ -0,0 +1,27 @@
+import { getOsShortcutSeparator } from '../getOsShortcutSeparator';
+
+describe('getOsShortcutSeparator', () => {
+  let userAgentSpy: jest.SpyInstance;
+
+  beforeEach(() => {
+    userAgentSpy = jest.spyOn(window.navigator, 'userAgent', 'get');
+  });
+
+  afterEach(() => {
+    userAgentSpy.mockRestore();
+  });
+
+  it('should return space for Windows', () => {
+    userAgentSpy.mockReturnValue(
+      'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0',
+    );
+    expect(getOsShortcutSeparator()).toBe(' ');
```

Comment:
What led you to choose a space instead of "+"? If I had to share a shortcut, I would instinctively write "Ctrl+C" instead of "Ctrl+C." I haven't used Windows for a long time so that I may be wrong in my assumptions.